impeller/core/texture_descriptor.h

```diff
@@ -46,6 +46,7 @@ struct TextureDescriptor {
       static_cast<TextureUsageMask>(TextureUsage::kShaderRead);
   SampleCount sample_count = SampleCount::kCount1;
   CompressionType compression_type = CompressionType::kLossless;
+  bool ignore_cache = false;
```

Comment:
See note above, this is temporary

Reply:
(Add TODO here as well linked to same issue)